ProjectSidewalk/SidewalkWebpage
# 3107: Showing mission screens after mission completion

Resolves #3107 

We were not showing the new mission screens after a validate mission was completed which is now fixed in this PR.

##### Before/After screenshots (if applicable)
NA.

##### Testing instructions
1. Start a validate mission.
2. Make sure you see the new mission screens before the mission.
3. Complete the mission.
4. Click on 'Validate more labels'. 
5. Make sure that mission screens are shown again.
6. Make sure that the mission screen label type corresponds with the mission's label type.

##### Things to check before submitting the PR 


These are NA:
